apps/test/unit/lib/kits/maker/boards/circuitPlayground/CircuitPlaygroundTestHelperFunctions.js

```diff
@@ -0,0 +1,50 @@
+import five from '@code-dot-org/johnny-five';
+import Playground from 'playground-io';
+import sinon from 'sinon';
+
+/**
+ * Simulate a raw value coming back from the board on the given component's pin.
+ * @param {five.Sensor|five.Thermometer} component
+ * @param {number} rawValue - usually in range 0-1023.
+ * @throws if nothing is monitoring the given analog pin
+ */
+export function setSensorAnalogValue(component, rawValue) {
+  const {board, pin} = component;
+  const readCallback = board.io.analogRead.args.find(
+    callArgs => callArgs[0] === pin
+  )[1];
+  readCallback(rawValue);
+}
+
+/**
+ * Generate a test board object.
+ * From rwaldron/johnny-five's newBoard() test helper:
+ * https://github.com/rwaldron/johnny-five/blob/dd47719/test/common/bootstrap.js#L83
+ * @returns {*}
```

Comment:
Sorry one other nit, think this could be updated to `@returns {five.Board}`?

Reply:
Sure - good call!